.circleci/config.yml

```diff
@@ -28,17 +28,10 @@ store-log-artifacts: &store-log-artifacts
     path: log
     destination: log
 
-flow-type-key: &flow-typed-cache-key flow-typed-{{ checksum "yarn.lock" }}-5
```

Comment:
do we want to cache TS compilation results or something?

Reply:
I don't think so. What was cached before were the typings, not actual code. Typescript's typings are stored inside `node_modules` so they are already cached.

Reply:
sounds good